docs/FAQ.md

```diff
@@ -2,7 +2,12 @@
 
 ## General
 
-- Why am I hitting "Cannot create an instance of type... " error?
+#### What provider cmdlets supported in SHiPS?
```

Comment:
Should be - "**Which** provider cmdlets **are** supported **with** SHiPS?"
- Should we add a link to the issues where folks are asking for additional cmdlets?

Reply:
Updated. The link exists from Readme